main.js

```diff
@@ -0,0 +1,12 @@
+let removeAPost = document.querySelector("header nav ul li").remove();
+
+let removeFeature = document.querySelector(".featured span").remove();
```

Comment:
```suggestion
document.querySelector(".featured span").remove();
```
`.remove` doesn't have a **return value** so saving this whole thing in a variable is unnecessary and might be confusing later. If you try to use the variable `removeFeature` later, it won't have any value at all; it won't be storing what you'd expect it to. `.remove` is destructive, so just using `document.querySelector("header nav ul li").remove();` will remove the element that you're targeting.